python/tests/conftest.py

```diff
@@ -121,12 +121,23 @@ def moto_s3_uri_incl_bucket(moto_s3_endpoint_and_credentials):
     ] + ":" + bucket + "?access=" + aws_access_key + "&secret=" + aws_secret_key + "&port=" + port
 
 
-@pytest.fixture(scope="function")
-def arctic_library(moto_s3_uri_incl_bucket):
-    ac = Arctic(moto_s3_uri_incl_bucket)
+@pytest.fixture(scope="function", params=("S3", "LMDB"))
```

Comment:
I only discovered this nice pytest feature this morning. It will effectively create one test run for each value of params for each test that uses the fixture. Therefore all tests that use the `arctic_library` fixture will now run against LMDB as well as S3.

Reply:
Neat!